.gitignore

```diff
@@ -81,3 +81,6 @@ Dockerfile
 # Generated documents for testing
 /result.csv
 /result.md
+/docs/pages/**/*.md
```

Comment:
Do we exclude these because the `instructions.md`s come from elsewhere (`/abis_mapping/templates`) ?
If yes , shouldn't we be more specific and put instructions.md in the exclude rule. We might want to keep other `.md` than `index.md` in the `/docs` folder.

Reply:
i wanted to exclude it because the instruction page can be named anything at the moment. In the github action it will create a file called incidental_occurrence_data_v2.md.
I thought for now we could leave it at this and as the other templates get their instructions generated we will have more of an idea how to best structure this better.

Reply:
ok